### PR TITLE
net-misc/urlscan: new package

### PR DESCRIPTION
**Pull request by mortebrume**
Package-Manager: Portage-2.3.53, Repoman-2.3.12
Signed-off-by: Andrew Attali <me@aphypnise.eu>

### CODE REVIEW

File: net-misc/urlscan/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      <name>Proxy Maintainers</name>
+    </maintainer>
+    <longdescription>
+      Urlscan is a small program that is designed to integrate with the "mutt"
+      mailreader to allow you to easily launch a Web browser for URLs contained
+      in email messages. It is a replacement for the "urlview" program.
+    </longdescription>
+    <upstream>
+      <maintainer>
+        <name>Scott Hansen</name>
```
Comment (juippis) [inline net-misc/urlscan/metadata.xml]: Do you have his consent on being added here? It's also in the wrong place and missing the most important part: e-mail.

File: net-misc/urlscan/urlscan-0.9.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Gentoo Authors
```
Comment (juippis) [inline net-misc/urlscan/urlscan-0.9.1.ebuild]: Why don't you use the default header? Commit with repoman to fix this automatically.

File: net-misc/urlscan/urlscan-0.9.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Permit to launch URLs to webrowser inside Mutt mail client."
```
Comment (juippis) [inline net-misc/urlscan/urlscan-0.9.1.ebuild]: No `.` in description please.

File: net-misc/urlscan/urlscan-0.9.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+HOMEPAGE="https://github.com/firecat53/urlscan"
+SRC_URI="https://github.com/firecat53/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="dev-python/urwid[${PYTHON_USEDEP}]"
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
+${RDEPEND}"
```
Comment (juippis) [inline net-misc/urlscan/urlscan-0.9.1.ebuild]: Mis-indent. Also, I doubt urwid is needed in DEPEND. However, setuptools seems to be required in RDEPEND. Here's what I suggest: 
```
DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
RDEPEND="${DEPEND}
	dev-python/urwid[${PYTHON_USEDEP}]"
```

File: net-misc/urlscan/urlscan-0.9.1.ebuild [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+DOCS=( COPYING README.rst )
```
Comment (juippis) [inline net-misc/urlscan/urlscan-0.9.1.ebuild]: These are part of the default `einstalldocs` array.

File: net-misc/urlscan/urlscan-0.9.1.ebuild [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+python_prepare_all () {
```
Comment (juippis) [inline net-misc/urlscan/urlscan-0.9.1.ebuild]: Well, this works, but if you want to reduce text it can be done in `src_prepare` too.

**Comment by juippis** [comment]
If you want to become a proxy-maintainer, you also need to have a bugzilla account with the maintainer e-mail (to assign bugs etc).

File: net-misc/urlscan/metadata.xml (hunk 1 above)
Comment (aphypnise) [reply net-misc/urlscan/metadata.xml]: Removed it, since it don't have his consent.

File: net-misc/urlscan/urlscan-0.9.1.ebuild (hunk 2 above)
Comment (aphypnise) [reply net-misc/urlscan/urlscan-0.9.1.ebuild]: It was generated from the ebuild emacs mode, and I always repoman + repoman commit so it's strange that it doesn't picked up. Fixed.

File: net-misc/urlscan/urlscan-0.9.1.ebuild (hunk 5 above)
Comment (aphypnise) [reply net-misc/urlscan/urlscan-0.9.1.ebuild]: Removed.

File: net-misc/urlscan/urlscan-0.9.1.ebuild (hunk 4 above)
Comment (aphypnise) [reply net-misc/urlscan/urlscan-0.9.1.ebuild]: Updated, thanks.

File: net-misc/urlscan/urlscan-0.9.1.ebuild (hunk 2 above)
Comment (juippis) [reply net-misc/urlscan/urlscan-0.9.1.ebuild]: Hmm, then I guess it wasn't technically wrong. Just something I haven't seen before, sorry :)

File: net-misc/urlscan/urlscan-0.9.1.ebuild [hunk 7]
```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
```
Comment (juippis) [inline net-misc/urlscan/urlscan-0.9.1.ebuild]: Sorry it's taken forever, but please remove support python3.4 and rebase against fresh copy of master repo.

File: net-misc/urlscan/urlscan-0.9.1.ebuild [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+DESCRIPTION="Permit to launch URLs to webrowser inside Mutt mail client"
+HOMEPAGE="https://github.com/firecat53/urlscan"
+SRC_URI="https://github.com/firecat53/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```
Comment (juippis) [inline net-misc/urlscan/urlscan-0.9.1.ebuild]: with EAPI-7 setuptools is a BDEPEND, and provided by distutils-r1.eclass. If you need it in RDEPEND, add it there without inheriting DEPEND :)

**Comment by ajakk** [comment]
Looks like this hasn't been touched in a long time, and since proxy-maint doesn't take new packages anymore I'm closing this. If still interested, please address juippis's concerns and add to [GURU](https://wiki.gentoo.org/wiki/Project:GURU)!

**Pull request closed without merging**